docs/Controls.md

```diff
@@ -4,7 +4,7 @@
 
 When armed the aircraft is ready to fly and any motors attached will spin when throttle is applied.
 
-By default arming and disarming is done using stick positions.  This is disabled when using a switch to arm.
+By default arming and disarming is done using stick positions.  Alternatively, you can use a switch to arm.
```

Comment:
I think you're right that alternatively doesn't relay mutual exclusion. I've reverted this to the previous wording now, explicitly specifying what 'this' means. Commit 2bdd821..dfa2684